### DISCUSSION TITLE
Does the produce action update the company name?

### DISCUSSION THREAD

**Discussion by AmirAlahmedy** [fastlane/fastlane#20136] [Q&A]
The [fastlane docs](https://docs.fastlane.tools/actions/produce/) states that passing the `company_name` to the `produce` action is only mandatory in the first build of the app. But what happens if it is passed with every build but with a different value? Is it going to replace the old company name?

**Comment by lucgrabowski** [comment]
It shouldn't do anything. It is required for the first build of the first app on developer account to set company name for the developer account. If you want to change it later, you need to contact Apple's support.

**Reply by AmirAlahmedy** [reply]
But from my experience, it does update the old company name. 
@lucgrabowski

**Reply by AmirAlahmedy** [reply]
The last time I tried this scenario was in Apr or Mar and it updated the company name. I'm not sure about the behavior now.
